privacyidea/lib/eventhandler/tokenhandler.py

```diff
@@ -45,6 +45,8 @@
 import json
 import logging
 import datetime
+from dateutil.parser import parse as parse_date_string
```

Comment:
``dateutil.parser.parse`` is pretty cool because it supports a lot of formats, but this could also cause problems in the future: If, e.g., a datetime string in the ``DD/MM/YYYY`` format is stored in the database for whatever reason, we once again run into the ``MM/DD/YYYY`` confusion, but we wouldn't notice, because ``parse`` just parses it! Because of that, I'm wondering if we should maybe build a wrapper that validates that the date string is in an expected format first and gives a warning if it isn't.